addons/editor-compact/userscript.js

```diff
@@ -0,0 +1,15 @@
+export default async function ({ addon, global, console }) {
+  // The workspace needs to be manually resized via a window resize event
+  // whenever the addon modifies or stops modifying UI elements
+  resizeWorkspace();
+
+  let resizeObserver = new ResizeObserver(resizeWorkspace);
+  let menuBar = await addon.tab.waitForElement(
+    '[class*="gui_menu-bar-position"][class*="menu-bar_menu-bar"][class*="box_box"]'
+  );
+  resizeObserver.observe(menuBar);
```

Comment:
Isn't `while(true)` necessary here? Isn't this element re-created every time the user enters the editor?